L1Trigger/L1TGlobal/plugins/L1TGlobalProducer.cc

```diff
@@ -49,6 +49,8 @@ void L1TGlobalProducer::fillDescriptions(edm::ConfigurationDescriptions& descrip
       ->setComment("InputTag for Calo Trigger Jet (required parameter:  default value is invalid)");
   desc.add<edm::InputTag>("EtSumInputTag", edm::InputTag(""))
       ->setComment("InputTag for Calo Trigger EtSum (required parameter:  default value is invalid)");
+  desc.add<edm::InputTag>("ZdcEtSumInputTag", edm::InputTag(""))
+      ->setComment("InputTag for ZDC EtSums Plus and Minus (required parameter:  default value is invalid)");
```

Comment:
I have a comment on the naming scheme: I think it's worth getting the names right (renaming parameters once they enter the HLT menu can be complicated), and here I would find it more natural to have `EtSumZdcInputTag` rather than `ZdcEtSumInputTag`. This also applies to other parts of the PR (on a minor note, using something like `EnergySumZdcTemplate` rather than `ZdcEnergySumTemplate` also helps "showing" that file close to the analogous `EnergySumTemplate`). I understand this is subjective, and it's not a strong opinion (clearly it's not a requirement, not that I can impose any).

Reply:
Ok @missirol I am working on the renaming.

Reply:
@elfontan and I have renamed globally - should now generally by etSumZdc or energySumZdc in latest commit